src/proto/single/mod.rs

```diff
@@ -108,26 +184,19 @@ impl Job {
         S: Into<String>,
         A: Into<serde_json::Value>,
     {
-        use rand::{thread_rng, Rng};
-        let random_jid = thread_rng()
-            .sample_iter(&rand::distributions::Alphanumeric)
-            .map(char::from)
-            .take(16)
-            .collect();
-        use chrono::prelude::*;
         Job {
```

Comment:
To avoid duplicating code here, can we now make this just call `JobBuilder::new(kind).args(args).build()`?